src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.TargetFrameworkInference.targets

```diff
@@ -46,6 +46,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- This is needed for the Appx targets -->
+    <DefaultLanguage Condition="'$(DefaultLanguage)' == ''">en-US</DefaultLanguage>
+    
+    <!-- Many of the targets use the VS Version in the path -->
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == '' or '$(VisualStudioVersion)' &lt; '14.0' ">15.0</VisualStudioVersion>
```

Comment:
Most of these targets require full MSBuild and will only work on Windows. The Xamarin ones should work on VSfM though long as it builds on Mono. That variable is for the XAML targets and those only work on Windows (for building UWP apps).